lib/kwtool_web/controllers/upload_controller.ex

```diff
@@ -0,0 +1,10 @@
+defmodule KwtoolWeb.UploadController do
+  use KwtoolWeb, :controller
+
+  def index(conn, _params) do
+    render(conn, "index.html")
+  end
+
+  def create do
```

Comment:
Why this endpoint is empty? 🤔

Reply:
This is for the backend ticket, for handling when the upload form submit.

I added it because it used in the form action of the template. `lib/kwtool_web/templates/upload/index.html.eex`.